test/integration/config_1/example_spec.rb

```diff
@@ -25,6 +25,7 @@
   its('content') { should match(/bind 0.0.0.0:8081/) }
   its('content') { should match(/bind 0.0.0.0:8180/) }
   its('content') { should match(/backend servers/) }
+  its('content') { should match(/default_backend servers/) }
```

Comment:
These two tests are redundant.  Both of them will match on the same line in the config, I think the second one that matches against the full `default_backend servers` line would be better.